package.json

```diff
@@ -6,7 +6,9 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^1.3.4",
     "react": "^18.2.0",
+    "react-bootstrap": "^2.7.2",
```

Comment:
You have react-bootstrap, but I notice you did not install (vanilla) bootstrap

Reply:
Good catch, I have added it in the latest commits.